tutorials/migrations/0001_initial.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.24 on 2019-12-04 16:50
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+import markdownx.models
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='Tutorial',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('title', models.TextField()),
+                ('html', models.TextField()),
+                ('markdown', markdownx.models.MarkdownxField()),
```

Comment:
Shouldn't this be displayed as a markdown editor? In my browser it shows a plain-text editor only when adding a new tutorial:
![image](https://user-images.githubusercontent.com/7835111/73636940-5b909800-4667-11ea-967c-fc3561b36dad.png)

Reply:
When writing markdown formatted text into the input field, a preview of the result appears below the text field. No tools for text editing are included so far.

Reply:
![image](https://user-images.githubusercontent.com/38939526/73742746-61f93f80-474d-11ea-8d76-4f6dca833b26.png)

Reply:
If this is not what you would expect from a markdown editor I could try to include this [package](https://github.com/agusmakmun/django-markdown-editor) instead of django-markdownx. The editor looks more complete and offers some text editing features.

Reply:
Yes, I think this would be the friendliest solution for our users. Question is, whether it is compatible with Django3.x

Reply:
The readme says django>=2.0 and there is also a ci pipeline for django 3. I test this and let you know :)

Reply:
@jh-RLI Did you integrate `django-markdown-editor` yet?

Reply:
@MGlauer  unfortunately I haven't had time yet but this is my next task!

Reply:
@MGlauer I could successfully install the editor. By the way, I noticed that the package is not called `django-markdown-editor` but `martor`